.github/workflows/lint_code.yml

```
@@ -0,0 +1,36 @@
+name: Lint
+
+on:
+  push:
+
+env:
+  TERRAFORM_VERSION: "1.3.6"
+
+jobs:
+  linting:
+    name: Lint Terraform files
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.10.0
```

there is a newer version of the action, you can consider using it
```suggestion
        uses: styfle/cancel-workflow-action@0.11.0
```

---

Thanks. This one was upgraded here: https://github.com/Nihisil/nimble-devops-ic-infrastructure/pull/18/commits/72364cbdceb6fb5034419ae01da350d69294b53b